features/hidden-until-found.yml

```
@@ -0,0 +1,7 @@
+name: hidden="until-found"
+description: 'The `hidden="until-found"` attribute TODO.'
```

I think this is how it's supposed to work?

```suggestion
description: 'The `hidden="until-found"` attribute hides an element until it is found using the browser's find-in-page search or it is directly navigated to by following a URL fragment.
```

---

Applied, I think this is accurate!